resources/lib/library.py

```diff
@@ -1500,9 +1501,9 @@ def explorer( self, history, location, label, thumbnail, itemType, isWidget = Fa
         widgetType = None
         addonType = None
 
-        dialogLabel = label[0].replace( "  >", "" )
+        dialogLabel = try_decode( label[0] ).replace( "  >", "" )
         if len( label ) != 1:
-            dialogLabel = label[0].replace( "  >", "" ) + " - " + label[ len( label ) - 1 ].replace( "  >", "" )
+            dialogLabel = try_decode( label[0] ).replace( "  >", "" ) + " - " + try_decode( label[ len( label ) - 1 ] ).replace( "  >", "" )
 
```

Comment:
`label` is actually an array of strings, so it's not removing the last char, it's getting the last element of the array. Probably still not doing it the pythonic way, though ;)